src/blend-impl.ts

```diff
@@ -2,53 +2,80 @@ import {
   abs,
   add,
   div,
-  float,
+  FLOAT0,
```

Comment:
Using these `FLOAT*` constants is NOT mandatory, but will re-use the same underlying AST leaf node and so a bit more memory friendly...